src/Kuzzle.js

```diff
@@ -47,6 +33,19 @@ class Kuzzle extends KuzzleEventEmitter {
     }
     this.protocol = protocol;
 
+    this._eventList = [
```

Comment:
I think you could have kept it in the node module scope with

```javascript
const		
   events = [		
     'connected',		
     'discarded',		
     'disconnected',		
     'loginAttempt',		
     'networkError',		
     'offlineQueuePush',		
     'offlineQueuePop',		
     'queryError',		
     'reconnected',		
     'tokenExpired'		
   ];
```

and simply implement a getter like 

```javascript
 get eventList () {
     return events
   }
```

without the need to modify anything else.

I know the use case of having multiple instance of the Kuzzle object is low, but this way, the memory fingerprint could be lighter since the array is not implemented for each instance.

Reply:
Yes you're right !
I did it like that because it's more logical to have the event list inside the kuzzle instance and not in the module global scope. In the futur we might want to let developer use middleware with the SDK and these middleware could add new events for example.